docs/setting-up.md

```diff
@@ -46,7 +46,7 @@ More information can be found at [this documentation](https://help.github.com/ar
 1. Install Google Cloud SDK. Follow the directions given [here](https://cloud.google.com/sdk/downloads).
    Note that you *do not* need to [initialize the SDK](https://cloud.google.com/sdk/docs/initializing).
    ```sh
-   # This command is to be run at the Google Cloud SDK directory
+   # Execute this command at the Google Cloud SDK directory
```

Comment:
Lets keep this consistent with line 61.

Change to ` Run the following command`